packages/core/src/modules/vc/W3cCredentialService.ts

```diff
@@ -103,17 +103,18 @@ export class W3cCredentialService {
    */
   public async verifyCredential(
     agentContext: AgentContext,
-    options: VerifyCredentialOptions,
-    verifyRevocationState = true
+    options: VerifyCredentialOptions
   ): Promise<W3cVerifyCredentialResult> {
+    options.verifyRevocationState = options.verifyRevocationState ?? true
```

Comment:
It's usually bad pratice to mutate input objects. Maybe we assign it as separate variable instead of mutating the input object?